src/app/pages/system/reporting/reporting.component.ts

```diff
@@ -65,22 +65,40 @@ export class ReportingComponent {
     public snackBar: MatSnackBar,
   ) {}
 
-
   afterInit(entityEdit: any) {
     this.entityForm = entityEdit;
+    this.rrd_checkbox = _.find(this.fieldConfig, {'name' : 'confirm_rrd_destroy'});
+    entityEdit.formGroup.controls['graph_age'].valueChanges.subscribe((res) => {
```

Comment:
1. need to check both `graph_age` and `graph_points` no matter which one triggered valueChanges event. 
2. one more suggestion, you are able to access the query result (original config) by `entityEdit.wsResponse`, so just need to compare the changed value with it :)

Reply:
Oh wow, good catch! Also, I'm glad to know about wsResponse, and I added a reset button using it